language/move-prover/spec-lang/src/ast.rs

```diff
@@ -135,6 +142,8 @@ impl std::fmt::Display for ConditionKind {
             Ensures => write!(f, "ensures"),
             Requires => write!(f, "requires"),
             RequiresModule => write!(f, "requires module"),
+            RequiresSpecCheck => write!(f, "assume"),
```

Comment:
This doesn't seem to be assume/assert. As this is used in error message, please use a more specific name.